taxi_service/urls.py

```diff
@@ -18,8 +18,12 @@
 from django.conf import settings
 from django.conf.urls.static import static
 
+from debug_toolbar.toolbar import debug_toolbar_urls
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls")),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
++ debug_toolbar_urls()
```

Comment:
The inclusion of `debug_toolbar_urls()` should be conditional based on the `DEBUG` setting to prevent exposing sensitive information in a production environment. Consider wrapping this in an `if settings.DEBUG:` block.